hack/run-e2e-test.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -uo pipefail
```

Comment:
Every other bash script in this repo sets:
```
set -o errexit
set -o nounset
set -o pipefail
```

Should we do the same here for consistency?

Reply:
Yes, we can do that. 
Added this in the latest commit